phobos/smurf/sensors.py

```diff
@@ -49,8 +49,9 @@ def get_refl_vars(self):
 class Joint6DOF(Sensor):
     def __init__(self, robot=None, name=None, link=None, **kwargs):
         super().__init__(robot=robot, name=name, joint=None, link=link, sensortype='Joint6DOF', **kwargs)
-        if self.link is None:
-            raise ValueError("Link used for sensor" + name + "does not exist!")
+        if not isinstance(link, representation.Link):
+            print(link)
+            raise AssertionError("Parsed invalid link")
```

Comment:
How about:
`raise AssertionError("Parsed invalid link:" + repr(link))`

Reply:
Is there a reason not to use f- strings ? Have seen this "+" concatenation all over the place